lib/modules/subredditInfo.js

```diff
@@ -89,7 +89,17 @@ function handleMouseOver(e: Event) {
 }
 
 async function showSubredditInfo(card, subreddit) {
-	const header = string.html`<div><a href="/r/${subreddit}">/r/${subreddit}</a></div>`;
+	const header = string.html`
+		<div class="subredditInfoToolTipHeader">
+			<a href="/r/${subreddit}" class="subredditTitleLink">/r/${subreddit}</a>
+		</div>`;
+
+	const makeSortLink = (subreddit, sort) =>
+		$('<a />').attr('href', `/r/${subreddit}/${sort}/`).addClass('subredditSortLink').text(sort);
+
+	['hot', 'new', 'controversial', 'top', 'rising'].map(sort => {
```

Comment:
Bit of a nitpick, but you should use `forEach` instead of `map` here. Map is intended for making a copy of an array with changes to each element.